tests/unit/providers/vector_io/test_vector_io_openai_vector_stores.py

```diff
@@ -86,10 +86,14 @@ async def test_register_and_unregister_vector_db(vector_io_adapter):
     assert dummy.identifier not in vector_io_adapter.cache
 
 
-async def test_query_unregistered_raises(vector_io_adapter):
+async def test_query_unregistered_raises(vector_io_adapter, vector_provider):
     fake_emb = np.zeros(8, dtype=np.float32)
-    with pytest.raises(ValueError):
-        await vector_io_adapter.query_chunks("no_such_db", fake_emb)
+    if vector_provider == "chroma":
```

Comment:
this is oddly specific (on the other hand one cannot just test for Exception since that is terrible also)...

Reply:
yeah agreed, if you have some suggestions happy to incorporate them too. 

I'll investigate making this cleaner in a follow up. Just want to make sure we can be confident about CI right now. 👍